src/openlcb/MemoryConfigClient.cxxtest

```diff
@@ -145,4 +152,22 @@ TEST_F(MemoryConfigLocalClientTest, readfromlocal)
     EXPECT_EQ(0, memcmp(&dataContents_[0], b->data()->payload.data(), dataContents_.size()));
 }
 
+TEST_F(MemoryConfigLocalClientTest, readfromlocalspecialspace)
+{
+    // In this test we read from the local node, to test that the
+    // service-client interaction is without any deadlock. We use a special
+    // space number (0xFD to 0xFF) to regression test a bug in the request
```

Comment:
What is the bug? It would be good for the comment to describe the bug or provide an issue number.

Reply:
There is no bug number. The bug is that responses for those space numbers were not routed to the client. I made a note of this.